src/i18n/index.js

```diff
@@ -0,0 +1,17 @@
+import arMessages from './messages/ar.json';
+import es419Messages from './messages/es_419.json';
+import frMessages from './messages/fr.json';
+import kokrMessages from './messages/ko_KR.json';
+import ptbrMessages from './messages/pt_BR.json';
```

Comment:
I think for now we only want to support `ar`, `es_419`, `fr`, `zh_CN`?

Reply:
[related?] When running the MFE locally, I'm seeing the following warnings from `@edx/frontend-platform`, a mix of "Unexpected locale" and "Missing locale" warnings:

<img width="448" alt="image" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/2828721/02d934d2-232c-437f-88a4-fdad0d934852">

Several of the "Unexpected locale" warnings are due to how Paragon includes those locales in the `paragonMessages` when `@edx/frontend-platform` doesn't have them listed in the `supportedLocales` ([source](https://github.com/openedx/frontend-platform/blob/5cf7cda8280d264785c46e75f1cbef98e0a1b9aa/src/i18n/lib.js#L38C7-L55)).

To document this issue, I've filed https://github.com/openedx/paragon/issues/3011 and will bring it to Paragon Working Group next week so Axim is aware of the issue.

Regarding the "Missing locale" warnings, I believe we should be resolving those warnings and included message files for each supported locale; IMHO, we should be consistent about what locales are officially supported by the Open edX platform, given this is an Open edX micro-frontend.

Reply:
If adding the locales to mitigate the "Missing locale" warnings, I believe you'd also want to include them in the list of `transifex_langs` in the `Makefile` ([source](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/fc42338979489ed55b0f215915eb3cbc486ba8dc/Makefile#L3)) as well?